### PR TITLE
DOC: extra details and warnings about WebServer

### PR DESCRIPTION
**Pull request by pieper**

### CODE REVIEW

File: Docs/user_guide/modules/webserver.md [hunk 1]
```diff
@@ -59,11 +69,7 @@ The web server can also be accessed via other commands such as `curl`. A `dict`
 For example, these commands may be used to download the MRHead sample data, change the screen layout and return a dictionary including the ID of the loaded volume:
 
 ```
-curl -X POST localhost:2016/slicer/exec --data "\
-import SampleData; volumeNode = SampleData.SampleDataLogic().downloadMRHead(); \
-slicer.app.layoutManager().setLayout(slicer.vtkMRMLLayoutNode.SlicerLayoutOneUpRedSliceView); \
-__execResult = {'volumeNodeID': volumeNode.GetID()} \
-"
+curl -X POST localhost:2016/slicer/exec --data "import SampleData; volumeNode = SampleData.SampleDataLogic().downloadMRHead(); slicer.app.layoutManager().setLayout(slicer.vtkMRMLLayoutNode.SlicerLayoutOneUpRedSliceView); __execResult = {'volumeNodeID': volumeNode.GetID()}"
```
Comment (lassoan) [inline Docs/user_guide/modules/webserver.md:72]: thank you, this change makes the example work on Windows

File: Docs/user_guide/modules/webserver.md [hunk 2]
```diff
@@ -44,11 +44,21 @@ Because the web server uses standard http, there are many off-the-shelf security
...
   - Log to GUI: Enable/disable the logging of messages in the module panel.
 
+:::{note}
+Logging to the console and/or the GUI is useful for learning about the software and for debugging, but slows down request handling and should be disabled for routine use.
+:::
+
+:::{warning}
+The `Slicer API exec` option exposes the full python interface of Slicer running with the same permissions as the Slicer app itself.  This means that users of that API can install arbtrary code on the system and execute it with the user's rights.  In practice this means that he user of the API can perform actions such as deleting files, sending emails, or insalling system software.  Exposing these capabilities is intentional as part of the design of the module, but users should be aware that enabling this feature is effectively the same as giving the user of the API the password to whatever account is running Slicer.
+
+Note also that even with the `Slicer API exec` disabled, it is possible that other endpoints expose vulerabilities such as buffer overruns that could lead to server exploits.  It is suggested that only trusted users be granted access to any of the API endpoints.
```
Comment (lassoan) [inline Docs/user_guide/modules/webserver.md:54]:
```suggestion
Note also that even with the `Slicer API exec` disabled, it is possible that other endpoints expose vulnerabilities such as buffer overruns that could lead to server exploits.  It is suggested that only trusted users be granted access to any of the API endpoints.
```

**Review by lassoan — DISMISSED** [review]
Thank you, it looks good to me. It makes sense to describe explicitly the risks associated with opening up any of these endpoints to the open internet.

File: Docs/user_guide/modules/webserver.md [hunk 3]
```diff
@@ -44,11 +44,21 @@ Because the web server uses standard http, there are many off-the-shelf security
   - Log to Console: Enable/disable the logging of messages in the console.
   - Log to GUI: Enable/disable the logging of messages in the module panel.
 
+:::{note}
+Logging to the console and/or the GUI is useful for learning about the software and for debugging, but slows down request handling and should be disabled for routine use.
+:::
+
+:::{warning}
+The `Slicer API exec` option exposes the full python interface of Slicer running with the same permissions as the Slicer app itself.  This means that users of that API can install arbitrary code on the system and execute it with the user's rights.  In practice this means that he user of the API can perform actions such as deleting files, sending emails, or installing system software.  Exposing these capabilities is intentional as part of the design of the module, but users should be aware that enabling this feature is effectively the same as giving the user of the API the password to whatever account is running Slicer.
```
Comment (jcfr) [inline Docs/user_guide/modules/webserver.md:52]:
```suggestion
The `Slicer API exec` option exposes the full python interface of Slicer running with the same permissions as the Slicer app itself.  This means that users of that API can install arbitrary code on the system and execute it with the user's rights.  In practice this means that the user of the API can perform actions such as deleting files, sending emails, or installing system software.  Exposing these capabilities is intentional and aligned with the design of the module, but users should be aware that enabling this feature is effectively the same as giving the user of the API the password to whatever account is running Slicer.
```

**Comment by jcfr** [comment]
If it looks good to everyone, I suggest we move forward with squashing & merging

**Comment by pieper** [comment]
Thank you for the edits 👍

**Pull request merged**